src/config.js

```diff
@@ -51,14 +51,15 @@ function version0To1 (oldData) {
         relation: 'parent',
         plugin: 'moneyd-uplink-xrp/node_modules/ilp-plugin-xrp-asym-client',
         assetCode: 'XRP',
-        assetScale: 6,
+        assetScale: 9,
```

Comment:
I think this should stay as 6; if someone is using a really old moneyd config then they're probably configured to use a connector with a scale of 6.

Reply:
That makes sense, will change back.